metatron-app/metatron-discovery
# F#1493 use json for map array

### Description
I've changed the internal structure Array and Map types from Java Objects to JSON strings.
Now, we can create an Array or Map (Object in a common word) types from String type values.

**Related Issue** : #1493 

### How Has This Been Tested?
1. Import & wrangle [json_in_column.csv.txt](https://github.com/metatron-app/metatron-discovery/files/3823084/json_in_column.csv.txt)
2. Click column1's context menu - Alter - Column type - Map
3. Click column1's context menu - Generate - Unnest
4. Click "Add" without any argument.
5. You'll see "A", "B_1" columns generated. (B_1 is because of name duplication)

![스크린샷 2019-11-08 오후 2 25 47](https://user-images.githubusercontent.com/42257727/68451827-adc24880-0233-11ea-8e1e-18e2381f8a7c.png)

#### Need additional checks?
Yes, please.

### Types of changes



### Checklist:
